src/plugins/syscalls/syscalls.cpp

```diff
@@ -299,6 +328,34 @@ static void print_footer(output_format_t format, uint32_t nargs)
     }
 }
 
+static event_response_t win_ret_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
+{
+    syscall_wrapper_t* wrapper = (syscall_wrapper_t*)info->trap->data;
+    syscalls* s = wrapper->sc;
+    int nargs = wrapper->wsc->num_args;
+
```

Comment:
Why not do sanity checking here to match that the return breakpoint was hit by the same process and thread as the original caller?